src/app/wishlists/wishlists.ts

```diff
@@ -38,14 +38,14 @@ let inventoryRolls: { [key: string]: InventoryWishListRoll } = {};
 export function getInventoryWishListRolls(
   allItems: DimItem[],
   rollsByHash: { [itemHash: number]: WishListRoll[] }
-): { [key: string]: InventoryWishListRoll } {
+): { [key: string]: InventoryWishListRoll } | undefined {
   if (
     !$featureFlags.wishLists ||
     _.isEmpty(rollsByHash) ||
```

Comment:
checking for empty `rollsByHash` (should) be equivalent to `wishListsEnabledSelector = ()=> (wishListsSelector(state)?.wishListAndInfo?.wishListRolls?.length || 0) > 0;`